src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/TdsParserHelperClasses.cs

```diff
@@ -72,7 +72,7 @@ internal enum TdsParserState
     /// <summary>
     /// Struct encapsulating the data to be sent to the server as part of Federated Authentication Feature Extension.
     /// </summary>
-    internal struct FederatedAuthenticationFeatureExtensionData
+    internal class FederatedAuthenticationFeatureExtensionData
```

Comment:
I see this `FederatedAuthenticationFeatureExtensionData` also `struct` in 
src\Microsoft.Data.SqlClient\netcore\src\Microsoft\Data\SqlClient\TdsParserHelperClasses.cs

Can we keep the consistency on both sides?